src/visuallyCompleteCalculator.ts

```diff
@@ -100,6 +100,14 @@ class VisuallyCompleteCalculator {
     this.activeMeasurementIndex = navigationIndex;
     Logger.info('VisuallyCompleteCalculator.start()', '::', 'index =', navigationIndex);
 
+    const activationStart: number =
+      // @ts-ignore
+      (performance?.getEntriesByType?.('navigation')[0]?.activationStart as number) || 0;
+
+    if (activationStart > 0) {
+      start = activationStart;
+    }
```

Comment:
I _think_ this should probably check if `activationStart > start`.  The reason being, if someone does an SPA navigation on a page that was prerendered, start should still be the manually supplied value, correct?

I think it would be nice if we can add one or two puppeteer test cases for this.  We can pair on that sometime if you like.